dags/example_astronauts.py

```diff
@@ -0,0 +1,67 @@
+"""
+This DAG queries the list of astronauts currently in space from the 
+Open Notify API and prints each astronaut's name and flying craft.
+
+There are two tasks, one to get the data from the API and save the results,
+and another to print the results. Both tasks are written in Python using
+Airflow's TaskFlow API, which allows you to easily turn Python functions into
+Airflow tasks, and automatically infer dependencies and pass data.
+
+The second task uses dynamic task mapping to create a copy of the task for
+each Astronaut in the list retrieved from the API. This list will change
+depending on how many Astronauts are in space, and the DAG will adjust 
+accordingly each time it runs.
+
+For more explanation and getting started instructions, see our Write your 
+first DAG tutorial: https://docs.astronomer.io/learn/get-started-with-airflow
+"""
+
+from airflow import Dataset
+from airflow.decorators import dag, task
+from pendulum import datetime
+import requests
+
+#Define the basic parameters of the DAG, like schedule and start date
+@dag(
+    schedule="@daily", start_date=datetime(2024, 1, 1), catchup=False, tags=["example"]
+)
```

Comment:
```suggestion
@dag(
    start_date=datetime(2024, 1, 1),
    schedule="@daily",
    catchup=False,
    doc_md=__doc__,
    tags=["example"],
)
```

Adding the doc string as DAG Docs :)